src/TBuffer.cpp

```diff
@@ -1791,7 +1791,7 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
                     mIgnoreTag = false;
                     mSkip.clear();
                     ch = '&';
-                } else if (mSkip == "&quot;" && ch == ';') {
+                } else if (mSkip == "&quot;" ) {
```

Comment:
You are already (*mis*- :stuck_out_tongue_winking_eye:) handling this in #2194 !  Trying to do the same thing in a another PR (this one) is not a good idea - if you are lucky when the other PR is merged in this change will be silently fast-forwarded over and ignored - but it can confuse the hell out of reviewers...!  To fix this now, put another commit into this PR to undo the change ***here*** and when we merge it in we will use the `squash and merge` option and it will magically disappear from *this* PR.  That will just leave you to fix the error in the other one...